javaee/ejb-client/src/main/java/org/camunda/bpm/application/impl/ejb/DefaultEjbProcessApplication.java

```diff
@@ -40,7 +39,7 @@
 @TransactionAttribute(TransactionAttributeType.REQUIRED)
 @ProcessApplication
 @Local(ProcessApplicationInterface.class)
-public class DefaultEjbProcessApplication extends EjbProcessApplication {
+public class DefaultEjbProcessApplication extends org.camunda.bpm.application.impl.EjbProcessApplication {
```

Comment:
❓Why is this change necessary?

Reply:
Does this break silently when changing back to import? 🤔

Reply:
A comments sounds feasible 👍

Reply:
Good idea in general, thanks for the feedback. Always helps to look at it with a fresh pair of eyes 👍 We have some other code like this that I will re-review to add comments as well.

Reply:
```suggestion
// Using fully-qualified class name instead of import statement to allow for automatic Jakarta transformation
public class DefaultEjbProcessApplication extends org.camunda.bpm.application.impl.EjbProcessApplication {
```